src/Glpi/Asset/AssetDefinition.php

```diff
@@ -593,9 +589,12 @@ public function getEnabledCapacities(): array
      *
      * @return array
      */
-    private function getDecodedCapacitiesField(): array
+    private function getDecodedCapacities(?string $encoded = null): array
     {
-        $capacities = @json_decode($this->fields['capacities'], associative: true);
+        if ($encoded === null) {
+            $encoded = $this->fields['capacities'];
+        }
+        $capacities = @json_decode($encoded);
```

Comment:
Why did you remove the `, associative: true` ?

Reply:
Because I want an object, not an array